app/assets/javascripts/code/services/_mobile_sessions.js

```diff
@@ -142,30 +148,25 @@ export const mobileSessions = (
       drawSession.undoDraw(session);
       session.markers = [];
 
-      const content = Session.averageValueAndUnit(session, selectedSensor);
-      let heatLevel = heat.levelName(Session.average(session));
+      let heatLevel = heat.levelName(Session.average(session, selectedSensor));
```

Comment:
Have you considered `const` instead of `let`?

Reply:
Also, maybe a `Heat` value (like `Session`) would allow us to clean up around more?